docs/changelog/2382.feature.rst

```diff
@@ -0,0 +1 @@
+On Windows, PROGRAMFILES, PROGRAMFILES(X86), and PROGRAMDATA environment variables are now passed through, unmasking system values necessary to locate resources such as a C compiler.
```

Comment:
Please make the env var names literals and remove the coma after windows

Reply:
Done